data/World/Fire Temple MQ.json

```diff
@@ -103,7 +161,12 @@
             "Fire Temple MQ GS Fire Wall Maze Center": "has_explosives",
             "Fire Temple MQ GS Above Fire Wall Maze": "
                 (can_use(Hookshot) and (Small_Key_Fire_Temple, 5)) or
-                (logic_fire_mq_above_maze_gs and can_use(Longshot))"
+                (logic_fire_mq_above_maze_gs and can_use(Longshot))",
+            "Fire Temple MQ Flame Maze Left Pot 1": "True",
+            "Fire Temple MQ Flame Maze Right Pot 1": "can_use(Hookshot) or logic_fire_mq_flame_maze",
+            "Fire Temple MQ Flame Maze Right Pot 2": "can_use(Hookshot) or logic_fire_mq_flame_maze",
+            "Fire Temple MQ Flame Maze Right Pot 3": "can_play(Song_of_Time) or Hover_Boots or logic_fire_mq_flame_maze",
+            "Fire Temple MQ Flame Maze Right Pot 4": "can_play(Song_of_Time) or Hover_Boots or logic_fire_mq_flame_maze"
```

Comment:
MQ Fire flame maze right pots 1 and 2 have different requirements from 3 and 4, and so should have different names.